### ISSUE TITLE
rpn_class_loss and rpn_class_logits

### ISSUE THREAD

**Issue by duanzhiihao** [jytime/Mask_RCNN_Pytorch#17]
There are two things in the `compute_rpn_class_loss()` which confuses me.
https://github.com/jytime/Mask_RCNN_Pytorch/blob/febe221020d6ecc4e3d0f708ddcf814da1566e0f/model.py#L979

First, `rpn_class_logits` is the FG/BG probabilities **Before** the softmax, as can be seen in the `RPN(nn.Module)`.
https://github.com/jytime/Mask_RCNN_Pytorch/blob/febe221020d6ecc4e3d0f708ddcf814da1566e0f/model.py#L855

Besides, `anchor_class` is a `torch.long` tensor.
https://github.com/jytime/Mask_RCNN_Pytorch/blob/febe221020d6ecc4e3d0f708ddcf814da1566e0f/model.py#L968
\
So why the cross-entropy can be computed between `a tensor which not necessarily in [0,1]` and `a torch.long tensor`?

Maybe I'm wrong or I miss some important things. Could anyone point out what's the case here?

**Comment by duanzhiihao** [comment]
This issue solved after I checked the docs of PyTorch.
